runner.py

```diff
@@ -20,7 +20,7 @@ def run(pytest, output_file, repeatUntilFailure, testDir, test_slice):
         log("Is going to repeat the test suite until failure")
     log("Preparing test suite with {}".format(pytest))
     testListFile = "test_list.txt"
-    collect_status = os.system('{} --collect-only {} > {}'.format(pytest, testDir, testListFile))
+    collect_status = os.system('python3 -m pytest -q --collect-only {} > {}'.format(testDir, testListFile))
```

Comment:
the previous version of this script was not compatible with the pytest updates from the 20.04 branch, no tests would ever be found.